src/main/java/com/epam/lab/page/ProfilePagePo.java

```diff
@@ -67,10 +86,32 @@ public PageElement getWallButton() {
     public PageElement getProfileView() {
         return profileView;
     }
+    
+    public void getAndClickAllContactsButton() {
```

Comment:
rename to clickAllContactsButtons

Reply:
can't you just return web element from this method?

Reply:
sorry